### PR TITLE
[BingSearch/News/Event] align to skill template

### PR DESCRIPTION
**Pull request by bobokids**
### Purpose
*What is the context of this pull request? Why is it being done?*
Align to skill template.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
. Bing search skill align to skill template
. News skill align to skill template
. Event skill align to skill template
. Phone skill change to share library

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/experimental/newsskill/Responses/Main/MainStringsTexts.lg [hunk 1]
```diff
@@ -35,4 +35,7 @@
 - What country or region are you in?
 
 # MarketRetryPrompt.Text(Data)
-- Couldn't find that country or region. What country or region do you want to search in?
\ No newline at end of file
+- Couldn't find that country or region. What country or region do you want to search in?
+
+# FeatureNotAvailable.Text(Data)
```
Comment (ssss141414) [inline skills/csharp/experimental/newsskill/Responses/Main/MainStringsTexts.lg]: Translate for other languages? Or maybe we can do this later.

File: skills/csharp/experimental/bingsearchskill/Adapters/DefaultAdapter.cs [hunk 2]
```diff
@@ -1,59 +1,127 @@
...
             {
-                CultureInfo.CurrentUICulture = new CultureInfo(context.Activity.Locale ?? "en-us");
-                await context.SendActivityAsync(templateManager.GenerateActivity(SharedResponses.ErrorMessage));
-                await context.SendActivityAsync(new Activity(type: ActivityTypes.Trace, text: $"Skill Error: {exception.Message} | {exception.StackTrace}"));
-                telemetryClient.TrackException(exception);
+                _telemetryClient.TrackException(exception);
 
-                if (context.IsSkill())
+                // Send a message to the user.
+                await turnContext.SendActivityAsync(_templateManager.GenerateActivityForLocale("ErrorMessage"));
```
Comment (xieofxie) [inline skills/csharp/experimental/bingsearchskill/Adapters/DefaultAdapter.cs]: Use SharedResponses.ErrorMessage instead

File: skills/csharp/experimental/eventskill/Adapters/DefaultAdapter.cs [hunk 3]
```diff
@@ -59,5 +59,72 @@ public class DefaultAdapter : BotFrameworkHttpAdapter
...
+        }
+
+        private async Task SendErrorMessageAsync(ITurnContext turnContext, Exception exception)
+        {
+            try
+            {
+                // Send a message to the user.
+                CultureInfo.CurrentUICulture = new CultureInfo(turnContext.Activity.Locale ?? "en-us");
+                await turnContext.SendActivityAsync(_templateManager.GenerateActivity(SharedResponses.ErrorMessage));
+                await turnContext.SendActivityAsync(new Activity(type: ActivityTypes.Trace, text: $"Phone Skill Error: {exception.Message} | {exception.StackTrace}"));
```
Comment (xieofxie) [inline skills/csharp/experimental/eventskill/Adapters/DefaultAdapter.cs]: Event Skill Error, not Phone

File: skills/csharp/experimental/newsskill/Responses/Main/MainStringsTexts.lg (hunk 1 above)
Comment (bobokids) [reply skills/csharp/experimental/newsskill/Responses/Main/MainStringsTexts.lg]: good catch. fixed.

File: skills/csharp/experimental/bingsearchskill/Adapters/DefaultAdapter.cs (hunk 2 above)
Comment (bobokids) [reply skills/csharp/experimental/bingsearchskill/Adapters/DefaultAdapter.cs]: fixed.

File: skills/csharp/experimental/eventskill/Adapters/DefaultAdapter.cs (hunk 3 above)
Comment (bobokids) [reply skills/csharp/experimental/eventskill/Adapters/DefaultAdapter.cs]: fixed.

**Pull request merged**